modules/cluster/outputs.tf

```
@@ -94,3 +94,9 @@ output "pipeline_viz_iam_role" {
   value       = module.iam_assumable_role_pipeline_visualizer.this_iam_role_name
   description = "The IAM Role that the pipeline visualizer pod will assume to authenticate"
 }
+
+output "ebscsi_addon_iam_role" {
```

There seem to be a typo here so it doesn't match the output of the parent module
```suggestion
output "ebs_csi_addon_iam_role" {
```

---

The new ebscsi role introduced is implemented with the TF module `iam-role-for-service-accounts-eks` and not `iam-assumable-role-with-oidc` like the others.  This TF module seems more straightforward to implement and don't know if there's limits rather thatn using the assumable-role version. If you have concerns about doing this please let me know. With this  different module I'm struggling with getting the correct syntax for the appropriate output variables.  For now I've removed the output statements under this branch (enable_ebs) and will create a different branch to test output var tests.  Not sure if this new ebs_csi role needs to have output variables defined. I can also test with `enable_worker_group = false` and attempt to documents these nuances